.gitlab-ci.yml

```diff
@@ -328,3 +386,29 @@ run_ram_slaves_testbench_sim:vivado2018.3:modelsim10.6c:
     - ipbb sim fli
     - ipbb sim make-project
     - ./vsim -c work.top -gIP_ADDR='X"c0a8c902"' -do 'run 1ms' -do 'quit'
+
+
+save_build_results:
+  stage: publish
+  image: gitlab-registry.cern.ch/ci-tools/ci-web-deployer:latest
+  only:
+    - /^pull-requests.[0-9]+$/
+    - master
+    - web
+  variables:
+    CI_OUTPUT_DIR: "build_results/"
+    OUTPUT_ROOT_DIR: "/eos/user/i/ipbusci/fw-gitlab-ci"
+  before_script:
+    - yum -y install openssh-clients
```

Comment:
openssh-clients should be included in the docker image

Reply:
Actually this does not apply as this job uses the IT-provided image and not the ipbus-fw-dev one. Ignore it